canton/community/app-base/src/main/scala/com/digitalasset/canton/environment/Environment.scala

```diff
@@ -9,14 +9,7 @@ import com.daml.grpc.adapter.ExecutionSequencerFactory
 import com.digitalasset.canton.DiscardOps
 import com.digitalasset.canton.concurrent.*
 import com.digitalasset.canton.config.*
-import com.digitalasset.canton.console.{
-  ConsoleEnvironment,
-  ConsoleGrpcAdminCommandRunner,
-  ConsoleOutput,
-  GrpcAdminCommandRunner,
-  HealthDumpGenerator,
-  StandardConsoleOutput,
-}
+import com.digitalasset.canton.console.{ConsoleEnvironment, ConsoleGrpcAdminCommandRunner, ConsoleOutput, GrpcAdminCommandRunner, HealthDumpGenerator, StandardConsoleOutput}
```

Comment:
Do you want to format to minimize future conflicts or you don't care ?

Reply:
The plan is: 
- merge this PR
- create a snapshot
- update daml dependences in canton (will fail)
- apply this change as a patch
- reformat
- update codedrop